debezium-core/src/main/java/io/debezium/data/Enum.java

```diff
@@ -24,22 +26,51 @@
      * 
      * @param allowedValues the comma separated list of allowed values; may not be null
      * @return the schema builder
+     * @deprecated Use {@link #builder(List)}
```

Comment:
No need to deprecate it, simply remove it. It's not a public API .

Reply:
I have decided against `Enum#builder(List)` and instead am making sure that the string being passed into the builder is formatted prior.